README.md

```diff
@@ -1,185 +1,217 @@
 
-## Installing / Getting started
+Most of the apps are built with [Express.js](https://expressjs.com/), and the front-end uses the [Nunjucks templating language](https://mozilla.github.io/nunjucks/templating.html) and the [GOV.UK Design System](https://design-system.service.gov.uk/).
 
-In order to get started you will need to run the LTS version of [Node.js](https://nodejs.org/en/).
+- [Getting Started](#getting-started)
+- [Tests](#tests)
+- [API Documentation](#api-documentation)
+- [Style Guide](#style-guide)
 
-Additionaly you can run the entire solution using Docker containers so a local [Docker](https://www.docker.com/products/docker-desktop) instance is required.
+## Getting Started
 
-The repository is structured like a monorepo with two main folders `apps` and `packages`.
+### Prerequisites
 
-`apps` - Holds the main Express.js applications responsible for the DB access API backend and the front facing web app.
-`packages` - Holds the common packages that can be used by all apps.
+* [Node.js](https://nodejs.org/en/) LTS & npm
+* [Docker](https://www.docker.com/products/docker-desktop) (to run a local db)
 
-## Developing
+### Database Setup
 
-### Built With
+#### Database Server Setup
 
-The entire solution is built on top [Express.js](https://expressjs.com/) and [Nunjucks templating language](https://mozilla.github.io/nunjucks/templating.html), both for the web and api backends.
+Use Docker to run an instance of SQL Server, replace "<YourStrong@Passw0rd>":
 
-### Prerequisites
+```shell
+docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=<YourStrong@Passw0rd>" -p 1433:1433 --name pins_sql_server --hostname pins_sql_server -d mcr.microsoft.com/mssql/server:2019-latest
+```
 
-Before you get started you need to make sure you are running the latest Node LTS version and latest NPM version.
+**Notes for M1 Macs**
 
-### Setting up Dev
+For M1 Macs until [this issue](https://github.com/microsoft/mssql-docker/issues/668) gets resolved you should use the following instructions, or as an alternative you can use [colima](https://github.com/abiosoft/colima) instead of Docker.
 
-Here's a brief intro about what a developer must do in order to start developing
-the project further:
+Additionally, SQL Client tools are not included in the sql-edge image for ARM64, so you won't be able to use sqlcmd from with the container. One workaround for this is to use the mcr.microsoft.com/mssql-tools docker image. You can run this temporarily in another container and connect to your `pins_sql_server` container by name if they are on the same network. Firstly, create a network:
```

Comment:
An alternative to running sql client docker images would be to run Azure Data Studio (https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall) and connect to the docker image

Reply:
Note added: https://github.com/Planning-Inspectorate/back-office/pull/874/commits/1e280801b6200916c0ca3eba880abac841920c12